vagrant-nsidc.yaml

```diff
@@ -1,7 +1,8 @@
+---
 project: nsidc_search
 gitrepo: git@github.com:nsidc/search-interface.git
 
 vsphere:
   template:
-    default: xenial64-nsidc-puppet4
-    ci: xenial64-docker-puppet4-jenkins1
+    default: bionic64-nsidc-puppet6
```

Comment:
This is still an old template; I think `jammy` is the most recent one (and uses puppet 7)

Reply:
Note that this story does not include updates to the branch v4.0.0-rc.10. The goal of the story is to get the code changes currently running in production back into `main`.